### PR TITLE
fixed issues #229, #501 -- cBioPortal column mouse over

### PR DESCRIPTION
**Pull request by jjgao**

### CODE REVIEW

**Comment by jjgao** [comment]
@onursumer this is about the cbioportal column in the patient view. Could you review the code?

File: core/src/main/java/org/mskcc/cbio/portal/servlet/PancancerMutationsJSON.java [hunk 1]
```diff
@@ -136,19 +139,31 @@ private static synchronized AccessControl getaccessControl() {
...
+                            CanonicalGene gene = daoGeneOptimized.getGene(symbol);
+                            if (gene!=null) {
+                                long entrezId = gene.getEntrezGeneId();
+                                
+                                // create the query string: (<ENTREZ ID>,<POSITION>)
+                                posWithEntrez.add("(" + entrezId + "," + position + ")");
+                            }
+                        }
+                        
 			String[] parts = proteinPos.split("_");
```
Comment (onursumer) [inline core/src/main/java/org/mskcc/cbio/portal/servlet/PancancerMutationsJSON.java]: Looks like this line of code is no longer needed

**Comment by onursumer** [comment]
We have another (duplicate) PancanMutationHistogram.js in the mutation-mapper library. I think we should try to refactor the patient view to use the mutation-mapper library before it is too late...

**Comment by jjgao** [comment]
Yes, we should consolidate the two tables. We can remove a lot of code in PancanMutationHistogram.js. The code to display thumbnails are not useful any more. Let's not handle that for this PR. I'll create another ticket for this.

**Pull request merged**